django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -409,6 +409,20 @@ span.clearable-file-input label {
     background: var(--accent);
 }
 
+.calendar caption a {
+	color: #333;
+}
+
+.calendar caption a:hover {
+	color: #000;
+	text-shadow: 0.1px 0.1px #000;
+}
+
+.calendar caption a:focus {
+	color: #000;
+	text-decoration: none;
+}
```

Comment:
This and above – based on screenshots and this code, there won’t be any indication that this text is clickable. We should change this – I’d recommend an underline that disappears on hover and focus.

Reply:
underline seems not true way.

Reply:
It doesn’t have to be an underline, but we do need to have _some_ way to indicate this text is interactive and not just displaying the month/year. The underline just happens to be the simplest option.

Other options could be to:

- Change the style of this header area so we can use the same blue link color as the "Today | Tomorrow" options below
- Add an icon next to each button (like https://duetds.github.io/date-picker/)